src/Sentry.Serilog/SentrySinkExtensions.cs

```diff
@@ -86,6 +88,7 @@ public static LoggerConfiguration Sentry(
         LogEventLevel? minimumEventLevel = null,
         IFormatProvider? formatProvider = null,
         ITextFormatter? textFormatter = null,
+        bool? experimentalEnableLogs = null,
```

Comment:
question: Order

I am super uncertain about the order here ... I am going back and forth.
I believe I am now a bit more leaning towards adding at the end, because adding it in the middle is a breaking change when invoked with positional arguments.

Reply:
If we do add it, I'd say add it at the end. That way people's code won't break when they bump to the new version.

> [!NOTE]
>   Given the ridiculous number of parameters to this method, we should consider refactoring it into some kind of Builder in future versions.